clients/python/tests/conftest.py

```diff
@@ -78,3 +130,23 @@ def mr_api(store_wrapper: MLMDStore) -> ModelRegistryAPIClient:
     mr = object.__new__(ModelRegistryAPIClient)
     mr._store = store_wrapper
     return mr
+
+
+def wait_for_grpc(container: DockerContainer, store: metadata_store.MetadataStore, timeout=6, interval=2):
+    start = time.time()
+    while True:
+        duration = time.time() - start
+        results = None
+        try:
+            results = store.get_contexts()
+        except errors.UnavailableError as e:
+            print("Container not ready. Retrying...")
+            print(e)
+            stdout = container.get_logs()
+            print(stdout)
```

Comment:
Nit: wouldn't this (or maybe another) order make more sense, as we're retrying after getting the error?

```suggestion
            print(e)
            print("Container logs:\n", container.get_logs())
            print("Container not ready. Retrying...")
```